pineappl_cli/src/plot.rs

```diff
@@ -99,32 +106,34 @@ fn map_format_channel(
         .join(" + ")
 }
 
+/// Convert channel contributions to Python tuples.
 fn map_format_channels(channels: &[(String, Vec<f64>)]) -> String {
     channels
         .iter()
         .map(|(label, bins)| {
             format!(
-                "                (r\"${}$\", np.array([{}]))",
+                "            (r\"${}$\", np.array([{}]))",
                 label,
                 map_format_e_join_repeat_last(bins)
             )
         })
         .join(",\n")
 }
 
+/// Convert PDF results to a Python tuple.
 fn format_pdf_results(pdf_uncertainties: &[Vec<Vec<f64>>], conv_funs: &[ConvFuns]) -> String {
     pdf_uncertainties
         .iter()
         .zip(conv_funs.iter().map(|fun| &fun.label))
         .map(|(values, label)| {
             format!(
-                "                (
+                "            (
                     \"{}\",
                     np.array([{}]),
                     np.array([{}]),
                     np.array([{}]),
                 ),",
-                label.replace('_', r"\_"),
```

Comment:
This will lead to problems once you use an NNPDF sets; these contain underscores which  will break LaTeX.

Reply:
no, I don't think so. 1. this leads to
```
/home/felix/Physik/N3PDF/PineAPPL/pineappl/test-data/plot2.py:525: SyntaxWarning: invalid escape sequence '\_'
  "NNPDF31\_nlo\_as\_0118\_luxqed",
```
and 2. I didn't check the actual plot script yet (since it is too complicated), but this would only be true if the label would be "tex-ified", which I think it does not. I'm testing atm with simple case in the unit test and there the plot (-label) is just fine

Reply:
Fixed in commit 68e73907cd8b30148ce3ab00bbee9087b1101971.